allensdk/test/brain_observatory/sync_utilities/test_sync_utilities.py

```diff
@@ -54,3 +54,17 @@ def test_get_synchronized_frame_times(monkeypatch, mock_dataset_fixture,
 
     obtained = su.get_synchronized_frame_times("dummy_path", sync_line_label_keys)
     assert np.allclose(obtained, expected)
+
+@pytest.mark.parametrize("mock_dataset_fixture,sync_line_label_keys,expected", [
+    ({"eye_tracking_timings": [0.020, 0.030, 0.040, 0.050, 3.0]},
+     Dataset.EYE_TRACKING_KEYS, [0.020, 0.030, 0.040, 0.050, 3.0]),
+
+    ({"behavior_tracking_timings": [0.080, 0.090, 0.100, 0.110, 8.0]},
+     Dataset.BEHAVIOR_TRACKING_KEYS, [0.08, 0.090, 0.100, 0.110, 8.0])
+], indirect=["mock_dataset_fixture"])
+def test_get_synchronized_frame_times_no_trim(monkeypatch, mock_dataset_fixture,
```

Comment:
👍  Please also add some cases where you expect a trim for completeness :)

Reply:
Those already exist in the form of the former unit tests, right? Should I add the `trim_after_spike=True` to them just to be clear?